netci.groovy

```diff
@@ -34,6 +34,10 @@ platformList.each { platform ->
     else if (os == 'Ubuntu') {
         buildCommand = "./build.sh --skip-prereqs --configuration ${configuration} --docker ubuntu.14.04 --targets Default"
     }
+	else if (os == 'Linux') {
+		os = 'Ubuntu16.04';
```

Comment:
I don't think this is correct.  Core-setup uses RHEL7.2 according to this comment:

https://github.com/dotnet/core-setup/blob/master/netci.groovy#L66-L69